File: reverseString.js
// Program to reverse the string "Web application"

function reverseString(str) {
    return str.split('').reverse().join('');
}

const originalString = "Web application";
const reversedString = reverseString(originalString);

console.log("Original String:", originalString);
console.log("Reversed String:", reversedString);

// Export the function for potential reuse
module.exports = reverseString;

File: ReverseString.java
public class ReverseString {
    public static void main(String[] args) {
        String word = "Hello Program";

        // Reverse using StringBuilder
        String reversed = new StringBuilder(word).reverse().toString();

        System.out.println("Original : " + word);
        System.out.println("Reversed : " + reversed);
    }
}
